.gitignore

```diff
@@ -7,3 +7,7 @@ NetCoreRestApi/BusinessLayer/obj/
 /NetCoreRestApi/WebAPI/bin/
 /NetCoreRestApi/WebAPI/obj/
 /NetCoreRestApi/WebAPI/WebAPI.csproj.user
+/NetCoreRestApi/DataLayer/obj
+/NetCoreRestApi/Common/bin/Debug/netcoreapp3.1
+/NetCoreRestApi/Common/obj
+/NetCoreRestApi/DataLayer/bin/Debug/netcoreapp3.1
```

Comment:
There should be another way how add all binary files so you don't need add new folders everytime

Reply:
done